src/store/tokens.ts

```diff
@@ -53,8 +58,8 @@ export const getters = getterTree(state, {
   getAllTokens(state): Tokens {
     return state.allTokens;
   },
-  getRestrictedTokens(state): Tokens {
-    return Object.fromEntries(Object.entries(state.allTokens).filter((e) => state.restrictedTokens.includes(e[1].symbol)));
+  getRestrictedTokens(state): TokenSymbol[] {
```

Comment:
You now have both `restrictedTokens` and `acceptableTokens`. I think it's an error-prone approach, since two lists may become diverged, which will cause different components (ones that use `restrictedTokens` and ones that use `acceptableTokens`) to be out of sync.

Do you need both though? From what I see, having `acceptableTokens` only should be enough.

Reply:
Totally agreed.
Let's just split this deployment and further refactoring.

There are couple of places where the code should be optimized, e.g. Transfer.vue / Deposit.vue